sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_client.py

```diff
@@ -211,7 +217,7 @@ def encrypt(self, algorithm, plaintext, **kwargs):
         :param algorithm: encryption algorithm to use
         :type algorithm: :class:`~azure.keyvault.keys.crypto.EncryptionAlgorithm`
         :param bytes plaintext: bytes to encrypt
-        :keyword bytes iv: optional initialization vector. For use with AES-CBC encryption.
+        :keyword bytes iv: optional initialization vector. Required for AES-CBC(PAD) encryption.
```

Comment:
`_validate_arguments` requires an iv for AES-CBC and raises when one's given for any other algorithm, so this isn't really optional--in all cases a caller either must, or must not, provide a value.